content/2015.07.cliquet.rationale-en.rst

```diff
@@ -0,0 +1,494 @@
+Why did we make Cliquet?
+########################
+
+:lang: en
+:url: why-cliquet
+:slug: post-cliquet-rationale
+:date: 2015-07-14
+:summary:
+    Based on Pyramid, Cliquet is a project that allows you to concentrate on
+    the essentials when building APIs.
+
+.. note::
+
+    Translated from the French by `Dan Phrawzty <https://twitter.com/phrawzty>`_.
```

Comment:
It's important to note when an article has been translated, imho. I would suggest that you add a similar note on the post you translated as well, yes.